src/System.EnterpriseServices/src/project.json

```diff
@@ -0,0 +1,9 @@
+{
+  "dependencies": {
+    "NETStandard.Library": "1.6.0-rc4-24214-03",
+    "Microsoft.Win32.Registry": "4.0.0-rc4-24214-03",
```

Comment:
Why are you referencing Registry in all these? It was referenced in the mscorlib one because the mscorlib facade has types from here. What really needs to be referenced in these facade project.json files is any preexisting contracts that contain any APIs from in these facades.